tadasets/swiss_holes_helpers.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+
+def in_a_hole(row, centers, radiuses):
+    """Helper function.
+
+    Parameters
+    ============
+
+    row: entry in matrix
+        One datapoint row.
+    centers: np.ndarray with shape (n_holes, d)
+        All the center coordinates of a ball.
+    radiuses: np.ndarray with shape (n_holes, )
+        All the radiuses of all the holes.
+    """
+    return any(np.apply_along_axis(np.linalg.norm,1,row - centers) <= radiuses) is False
+
+def eliminate_overlaps(centers, radiuses):
+    """Eliminates larger overlapping circles.
+    TBD: Could probably be sped up using the miniball algorithm.
+
+    Parameters
+    ============
+
+    centers: np.ndarray with shape (n_holes, d)
+        All the center coordinates of a ball.
+    radiuses: np.ndarray with shape (n_holes, )
```

Comment:
spelling radii?

Reply:
Haha yes, sorry, english is my second language :P

Reply:
Changed this now!